src/EntryForm.js

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import firebase from "./firebase";
+import {push, ref, getDatabase} from "firebase/database"
+import Swal from "sweetalert2";
+
+const EntryForm = () => {
+
+    const [title, setTitle] = useState("");
+    const [body, setBody] = useState("");
+
+    const date = new Date();
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const year = date.getFullYear();
+    const entryDate = `${day}-${month}-${year}`;
+
+
+    const handleBodyChange = (e) => {
+      setBody(e.target.value);
+    }
+    
+    const handleTitleChange = (e) => {
+      setTitle(e.target.value);
+    }
+  
+    const handleSubmit = (e) => {
+      e.preventDefault();
+
+      if (body && title) {
+        const database = getDatabase(firebase);
+        const dbRef = ref(database);
+        console.log(body, title, entryDate);
+  
+        const diaryEntry = {
+          content:body,
+          title:title,
+          date: entryDate,
+        }
+    
+        push(dbRef, diaryEntry);
```

Comment:
Using firebase here is amazing. You don't have to pass props around like a crazy person (like I did)!